featured/maven/maven.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: devops.kubesphere.io/v1alpha3
+kind: ClusterTemplate
+metadata:
+  name: maven
+  annotations:
+    devops.kubesphere.io/displayNameEN: "Maven"
+    devops.kubesphere.io/displayNameZH: "Maven"
+    devops.kubesphere.io/descriptionEN: "Designed for the continuous integration of most Maven projects, including dependency downloading, testing, building, and artifact archiving."
+    devops.kubesphere.io/descriptionZH: "专为大多数 Maven 项目的持续集成而设计，包括依赖下载、测试、构建和归档。"
+    devops.kubesphere.io/icon: "maven.svg"
+spec:
+  parameters:
+    - name: CloneURL
```

Comment:
This template only supports a git repository. How about changing the name to `GitRepository`?